<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWindow</class>
 <widget class="QWidget" name="SelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectWindow</string>
  </property>
  <widget class="QLabel" name="Mode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Modeï¼š</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hint">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hint:</string>
   </property>
  </widget>
  <widget class="QLabel" name="GridSize">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Size:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxHint">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxGridSize">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxMode">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Easy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hard</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Sudoku</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonGenerate</sender>
   <signal>clicked()</signal>
   <receiver>SelectWindow</receiver>
   <slot>on_pushButtonGenerate_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButtonGenerate_click()</slot>
 </slots>
</ui>
